### DISCUSSION TITLE
My domain seems to give invalid certificate error and doesn't seem to connect at all?

### DISCUSSION THREAD

**Discussion by v3vishal** [is-a-dev/register#15949] [Support] [answered]
I added the CNAME record as in the tutorial to my json file linking to my GitHub pages but it doesn't seem to work, despite the pull request being approved. Please help, I am not very knowledgeable regarding domains

**Comment by iostpa** [comment, marked as answer]
For the certificate error, you need a SSL certificate. Since you are using Github Pages you can go along the following steps: Go to your repo > Settings > Pages Section > Under where you put your domain there should be a field called "Enforce HTTPS" which you should check that.
If you followed the steps right then it should in a couple minutes be accessible.
The reason why you are getting this error is because our TLD (.dev) is a top level domain, which requires SSL certificates.

**Reply by v3vishal** [reply]
thanks! it works now :)
